packages/logger/src/logger-properties.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+export interface LoggerProperties {
+    scanId?: string;
+    batchRequestId?: string;
```

Comment:
Should ``batchRequestId`` be required? Aren't we always going to have it?

Reply:
There won't be one if we decide to log anything before reading the request documents or assigning an id or something like that. Would it be better to not have the field in that case, or have the field exist but be empty?